2019-08-29 16:03:31 INFO  Version:46 - HHH000412: Hibernate Core {5.4.2.Final}
2019-08-29 16:03:31 INFO  Version:49 - HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2019-08-29 16:03:31 WARN  pooling:72 - HHH10001002: Using Hibernate built-in connection pool (not for production use!)
2019-08-29 16:03:31 INFO  pooling:115 - HHH10001005: using driver [org.postgresql.Driver] at URL [jdbc:postgresql://localhost:5432/myapp]
2019-08-29 16:03:31 INFO  pooling:124 - HHH10001001: Connection properties: {user=myuser, password=****, pool_size=1}
2019-08-29 16:03:31 INFO  pooling:129 - HHH10001003: Autocommit mode: false
2019-08-29 16:03:31 INFO  DriverManagerConnectionProviderImpl:239 - HHH000115: Hibernate connection pool size: 20 (min=1)
2019-08-29 16:03:31 INFO  Dialect:158 - HHH000400: Using dialect: org.hibernate.dialect.PostgreSQLDialect
2019-08-29 16:03:32 INFO  JtaPlatformInitiator:52 - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2019-08-29 16:03:34 WARN  SqlExceptionHelper:137 - SQL Error: 0, SQLState: 23505
2019-08-29 16:03:34 ERROR SqlExceptionHelper:142 - ERROR: duplicate key value violates unique constraint "task_name_user_id_unique"
  Подробности: Key (user_id, task_name)=(111, Test) already exists.
2019-08-29 16:03:35 WARN  SqlExceptionHelper:137 - SQL Error: 0, SQLState: 23505
2019-08-29 16:03:35 ERROR SqlExceptionHelper:142 - ERROR: duplicate key value violates unique constraint "email_unique"
  Подробности: Key (user_email)=(test@gmail.com) already exists.
2019-08-29 16:03:35 WARN  SqlExceptionHelper:137 - SQL Error: 0, SQLState: 23505
2019-08-29 16:03:35 ERROR SqlExceptionHelper:142 - ERROR: duplicate key value violates unique constraint "task_name_user_id_unique"
  Подробности: Key (user_id, task_name)=(111, Test) already exists.
2019-08-29 16:03:35 WARN  SqlExceptionHelper:137 - SQL Error: 0, SQLState: 23505
2019-08-29 16:03:35 ERROR SqlExceptionHelper:142 - ERROR: duplicate key value violates unique constraint "email_unique"
  Подробности: Key (user_email)=(test@gmail.com) already exists.
2019-08-29 16:04:11 INFO  UserService:26 - Adding user
2019-08-29 16:04:11 INFO  Version:46 - HHH000412: Hibernate Core {5.4.2.Final}
2019-08-29 16:04:11 INFO  Version:49 - HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2019-08-29 16:04:11 WARN  pooling:72 - HHH10001002: Using Hibernate built-in connection pool (not for production use!)
2019-08-29 16:04:11 INFO  pooling:115 - HHH10001005: using driver [org.postgresql.Driver] at URL [jdbc:postgresql://localhost:5432/myapp]
2019-08-29 16:04:11 INFO  pooling:124 - HHH10001001: Connection properties: {user=myuser, password=****, pool_size=1}
2019-08-29 16:04:11 INFO  pooling:129 - HHH10001003: Autocommit mode: false
2019-08-29 16:04:11 INFO  DriverManagerConnectionProviderImpl:239 - HHH000115: Hibernate connection pool size: 20 (min=1)
2019-08-29 16:04:11 INFO  Dialect:158 - HHH000400: Using dialect: org.hibernate.dialect.PostgreSQLDialect
2019-08-29 16:04:12 INFO  JtaPlatformInitiator:52 - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2019-08-29 16:24:45 INFO  Version:46 - HHH000412: Hibernate Core {5.4.2.Final}
2019-08-29 16:24:45 INFO  Version:49 - HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2019-08-29 16:24:45 WARN  pooling:72 - HHH10001002: Using Hibernate built-in connection pool (not for production use!)
2019-08-29 16:24:45 INFO  pooling:115 - HHH10001005: using driver [org.postgresql.Driver] at URL [jdbc:postgresql://localhost:5432/myapp]
2019-08-29 16:24:45 INFO  pooling:124 - HHH10001001: Connection properties: {user=myuser, password=****, pool_size=1}
2019-08-29 16:24:45 INFO  pooling:129 - HHH10001003: Autocommit mode: false
2019-08-29 16:24:45 INFO  DriverManagerConnectionProviderImpl:239 - HHH000115: Hibernate connection pool size: 20 (min=1)
2019-08-29 16:24:45 INFO  Dialect:158 - HHH000400: Using dialect: org.hibernate.dialect.PostgreSQLDialect
2019-08-29 16:24:46 INFO  JtaPlatformInitiator:52 - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2019-08-29 16:24:48 INFO  TaskDaoHibernate:94 - Task was added successfully
2019-08-29 16:24:48 INFO  TaskDaoHibernate:58 - Get task by Name and Id
2019-08-29 16:24:48 INFO  TaskDaoHibernate:109 - Task was successfully deleted
2019-08-29 16:24:48 INFO  TaskDaoHibernate:81 - Get task by ID was FAILED
2019-08-29 16:24:48 INFO  TaskDaoHibernate:94 - Task was added successfully
2019-08-29 16:24:48 INFO  TaskDaoHibernate:58 - Get task by Name and Id
2019-08-29 16:24:48 INFO  TaskDaoHibernate:78 - Get task By Id
2019-08-29 16:24:48 INFO  TaskDaoHibernate:109 - Task was successfully deleted
2019-08-29 16:24:48 INFO  TaskDaoHibernate:94 - Task was added successfully
2019-08-29 16:24:48 INFO  TaskDaoHibernate:58 - Get task by Name and Id
2019-08-29 16:24:48 WARN  SqlExceptionHelper:137 - SQL Error: 0, SQLState: 23505
2019-08-29 16:24:48 ERROR SqlExceptionHelper:142 - ERROR: duplicate key value violates unique constraint "task_name_user_id_unique"
  Подробности: Key (user_id, task_name)=(107, Test) already exists.
2019-08-29 16:24:48 INFO  TaskDaoHibernate:97 - Task wasn't added (UNIQUE CONSTRAINT)
2019-08-29 16:24:48 INFO  TaskDaoHibernate:109 - Task was successfully deleted
2019-08-29 16:24:48 INFO  TaskDaoHibernate:94 - Task was added successfully
2019-08-29 16:24:48 INFO  TaskDaoHibernate:58 - Get task by Name and Id
2019-08-29 16:24:48 INFO  TaskDaoHibernate:78 - Get task By Id
2019-08-29 16:24:48 INFO  TaskDaoHibernate:109 - Task was successfully deleted
2019-08-29 16:24:48 INFO  UserDaoHibernate:91 - User was successfully added
2019-08-29 16:24:48 INFO  UserDaoHibernate:105 - User was successfully deleted
2019-08-29 16:24:48 INFO  UserDaoHibernate:91 - User was successfully added
2019-08-29 16:24:48 INFO  UserDaoHibernate:144 - User was successfully updated
2019-08-29 16:24:48 INFO  UserDaoHibernate:105 - User was successfully deleted
2019-08-29 16:24:48 INFO  UserDaoHibernate:91 - User was successfully added
2019-08-29 16:24:48 INFO  UserDaoHibernate:105 - User was successfully deleted
2019-08-29 16:24:48 INFO  UserDaoHibernate:91 - User was successfully added
2019-08-29 16:24:48 INFO  UserDaoHibernate:105 - User was successfully deleted
2019-08-29 16:24:48 INFO  UserDaoHibernate:91 - User was successfully added
2019-08-29 16:24:48 WARN  SqlExceptionHelper:137 - SQL Error: 0, SQLState: 23505
2019-08-29 16:24:48 ERROR SqlExceptionHelper:142 - ERROR: duplicate key value violates unique constraint "email_unique"
  Подробности: Key (user_email)=(test@gmail.com) already exists.
2019-08-29 16:24:48 INFO  UserDaoHibernate:94 - User wasn't successfully added
2019-08-29 16:24:48 INFO  UserDaoHibernate:105 - User was successfully deleted
2019-08-29 16:24:48 INFO  TaskDaoHibernate:94 - Task was added successfully
2019-08-29 16:24:48 INFO  TaskDaoHibernate:58 - Get task by Name and Id
2019-08-29 16:24:48 INFO  TaskDaoHibernate:109 - Task was successfully deleted
2019-08-29 16:24:48 INFO  TaskDaoHibernate:81 - Get task by ID was FAILED
2019-08-29 16:24:49 INFO  TaskDaoHibernate:94 - Task was added successfully
2019-08-29 16:24:49 INFO  TaskDaoHibernate:58 - Get task by Name and Id
2019-08-29 16:24:49 INFO  TaskDaoHibernate:78 - Get task By Id
2019-08-29 16:24:49 INFO  TaskDaoHibernate:109 - Task was successfully deleted
2019-08-29 16:24:49 INFO  TaskDaoHibernate:94 - Task was added successfully
2019-08-29 16:24:49 INFO  TaskDaoHibernate:58 - Get task by Name and Id
2019-08-29 16:24:49 WARN  SqlExceptionHelper:137 - SQL Error: 0, SQLState: 23505
2019-08-29 16:24:49 ERROR SqlExceptionHelper:142 - ERROR: duplicate key value violates unique constraint "task_name_user_id_unique"
  Подробности: Key (user_id, task_name)=(107, Test) already exists.
2019-08-29 16:24:49 INFO  TaskDaoHibernate:97 - Task wasn't added (UNIQUE CONSTRAINT)
2019-08-29 16:24:49 INFO  TaskDaoHibernate:109 - Task was successfully deleted
2019-08-29 16:24:49 INFO  TaskDaoHibernate:94 - Task was added successfully
2019-08-29 16:24:49 INFO  TaskDaoHibernate:58 - Get task by Name and Id
2019-08-29 16:24:49 INFO  TaskDaoHibernate:78 - Get task By Id
2019-08-29 16:24:49 INFO  TaskDaoHibernate:109 - Task was successfully deleted
2019-08-29 16:24:49 INFO  UserDaoHibernate:91 - User was successfully added
2019-08-29 16:24:49 INFO  UserDaoHibernate:105 - User was successfully deleted
2019-08-29 16:24:49 INFO  UserDaoHibernate:91 - User was successfully added
2019-08-29 16:24:49 INFO  UserDaoHibernate:144 - User was successfully updated
2019-08-29 16:24:49 INFO  UserDaoHibernate:105 - User was successfully deleted
2019-08-29 16:24:49 INFO  UserDaoHibernate:91 - User was successfully added
2019-08-29 16:24:49 INFO  UserDaoHibernate:105 - User was successfully deleted
2019-08-29 16:24:49 INFO  UserDaoHibernate:91 - User was successfully added
2019-08-29 16:24:49 INFO  UserDaoHibernate:105 - User was successfully deleted
2019-08-29 16:24:49 INFO  UserDaoHibernate:91 - User was successfully added
2019-08-29 16:24:49 WARN  SqlExceptionHelper:137 - SQL Error: 0, SQLState: 23505
2019-08-29 16:24:49 ERROR SqlExceptionHelper:142 - ERROR: duplicate key value violates unique constraint "email_unique"
  Подробности: Key (user_email)=(test@gmail.com) already exists.
2019-08-29 16:24:49 INFO  UserDaoHibernate:94 - User wasn't successfully added
2019-08-29 16:24:49 INFO  UserDaoHibernate:105 - User was successfully deleted
